### PR TITLE
Updated doc for swaggerUrls

### PR DESCRIPTION
**Pull request by vabhishek-me**
Added doc for loading multiple swagger docs using swaggerUrls property.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -153,6 +153,29 @@ var options = {
...
+var options = {
+  swaggerUrls: [
+    {
+      url: 'http://petstore.swagger.io/v2/swagger.json',
+      name: 'Spec1'
+    },
+    {
+      url: 'http://petstore.swagger.io/v2/swagger.json',
+      name: 'Spec2'
+    }
```
Comment (shockey) [inline README.md]: example value is missing a `]`

File: README.md [hunk 2]
```diff
@@ -153,6 +153,29 @@ var options = {
...
+}
```
Comment (shockey) [inline README.md]: My mental model is that top-level options affect the`swagger-ui-express` layer, and options within `swaggerOptions` affect the underlying UI. 

With that in mind, I think it'd make more sense to do this through `swaggerOptions` and preserve the UI's `urls` naming convention, i.e.:

```js
var options = {
  swaggerOptions: {
    urls: [
      {
        url: 'http://petstore.swagger.io/v2/swagger.json',
        name: 'Spec1'
      },
      {
        url: 'http://petstore.swagger.io/v2/swagger.json',
        name: 'Spec2'
      }
    ]
  }
}
```

**Comment by vabhishek-me** [comment]
I think we should make clear in docs this mental model. It's a good model I mean.
Also, the `swaggerUrl` option in the existing doc should also be updated to use `url` from `swaggerOptions`.
And maybe we can reference to https://github.com/swagger-api/swagger-ui/blob/master/docs/usage/configuration.md in `Custom swagger options`, so that people using this package know what else they can configure. 

I'll update the PR. See if it fits.

**Comment by vabhishek-me** [comment]
please check this now.
I've added a link to the options available in `swaggerOptions`.
Also, I've updated the existing document to use `url` and `urls` from `swaggerOptions`.

**Pull request merged**